protocol/x/listing/keeper/listing.go

```diff
@@ -23,3 +25,28 @@ func (k Keeper) GetMarketsHardCap(ctx sdk.Context) (hardCap uint32) {
 	k.cdc.MustUnmarshal(b, &result)
 	return result.Value
 }
+
+// Function to wrap the creation of a new clob pair
+// Note: This will only list long-tail/isolated markets
+func (k Keeper) CreateClobPair(
```

Comment:
nit: Happy path test?

Reply:
will be tested in aggregate at the top level PML msg. This function barely does anything